Tests/LogicTests/SNTFileInfoTest.m

```diff
@@ -132,6 +132,65 @@ - (void)testBundle {
   XCTAssertEqualObjects([sut bundlePath], @"/Applications/Safari.app");
 }
 
+- (void)testAncestorBundle {
+  NSString *nestedBundleBinary =
+      @"/Applications/Safari.app/Contents/XPCServices/"
+      @"com.apple.Safari.BrowserDataImportingService.xpc/"
+      @"Contents/MacOS/com.apple.Safari.BrowserDataImportingService";
```

Comment:
I realize a lot of these tests rely on system paths (which is bad anyway) but this one makes me nervous as I think it's likely to change in future releases. Could you create a bundle structure in the LogicTests/Resources folder and use that for this test?